lessons/codap-bar-charts/langs/en-us/pages/data-displays-organizer.adoc

```diff
@@ -0,0 +1,57 @@
+= Data Displays Organizer
+
+Put a check mark to indicate whether each chart listed below displays _1 variable or 2 variables_, and whether it displays data that is _categorical or numeric_. In the notes column, add any relevant reminders to yourself about when to use each display, how to create it, and how to interpret it.
+
+[.FillVerticalSpace, cols="^.^2,^.^3,^.^3,^.^3,^.^10", options="header"]
+|===
+|
+| 1 Variable, Categorical
```

Comment:
This might make more sense with  a column titled "What Variables?" with the options as checkboxes in each row...